rdr_service/api/participant_summary_api.py

```diff
@@ -53,8 +53,12 @@ def get(self, p_id=None):
                     raise Forbidden
             return super(ParticipantSummaryApi, self)._query("participantId")
 
-    def _make_query(self):
-        query = super(ParticipantSummaryApi, self)._make_query()
+    def _make_query(self, check_invalid=True):
+        pair_constraint, message = self._check_constraints()
+        if pair_constraint:
```

Comment:
nit: `pair_constraint` sounds like an object or something that can give information about a constraint (like what's required or what the constraint is for) rather than an indicator that the constraint has failed. What about `constraint_failed`?